### DISCUSSION TITLE
Can't login to kibana since so-elastic-auth

### DISCUSSION THREAD

**Discussion by secuser0965453** [Security-Onion-Solutions/securityonion#4696] [Unsupported Versions] [answered]
Hi,
I've upgraded my installation with soup this weekend, everything went well, no errors.
I've then ran "sudo so-elastic-auth true", changed my password according to doc and output, but my account isn't recognized by Kibana.
I still can login to SOC.
I've run a highstate for all nodes and it terminate without any error.
No errors in Kratos logs.
Please advise.
Thanks.

**Comment by jertel** [comment]
Hello - below are some actions you can take to investigate:

- Execute `so-elasticsearch-query /` on the manager - Does it succeed?
- Review logs in `/opt/so/log/kibana/`
- Review logs in `/opt/so/log/elasticsearch/`
- Disable Elastic auth via `so-elastic-auth false`. Can you access Kibana again as an anonymous user?
- Re-enable Elastic auth via `so-elastic-auth true`. Does is still list your user as needing to change the password?
- Review users listed in `/opt/so/conf/elasticsearch/users` file. Does it have the correct email addresses listed, with a hash beginning with `$2a`?

**Comment by secuser0965453** [comment, marked as answer]
Thanks for your help Jertel.
Already tried to disable and  re-enable Elastic auth.
After running soup today it's now working.
